cpp/src/plane_extraction.cpp

```diff
@@ -0,0 +1,569 @@
+#include <numeric>
+#include "deplex/plane_extraction.h"
+
+#ifdef DEBUG_CAPE
+#include <iostream>
+#include <fstream>
+#endif
+
+namespace deplex {
+
+const config::Config CAPE::__default_config{
```

Comment:
Why not to read it from file? If you have file configs for others, let's have also file for default